Makefile

```
@@ -60,6 +56,11 @@ default: help
 .env:
 	cp .env.dist .env
 
+git-hook-pre-commit:
+	ln -s -f ../../bin/git-hook-pre-commit .git/hooks/pre-commit
+	@echo "Git pre-commit hook linked"
+.PHONY: git-hook-pre-commit
```

Makefile rules  are usually file-path-based:

```suggestion
.git/hooks/pre-commit:
	ln -sf ../../bin/git-hook-pre-commit .git/hooks/pre-commit

git-hook-pre-commit:  ## Install git pre-commit hook
git-hook-pre-commit: .git/hooks/pre-commit
	@echo "Git pre-commit hook linked"
.PHONY: git-hook-pre-commit
```